nodejs/18/Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN         apt update \
             && apt -y install ffmpeg iproute2 git sqlite3 libsqlite3-dev python3 python3-dev ca-certificates dnsutils tzdata zip tar curl build-essential libtool iputils-ping libnss3 tini \
             && useradd -m -d /home/container container
 
-RUN         npm install npm@9.8.1 typescript ts-node @types/node --location=global
+RUN         npm install npm@10.x.x typescript ts-node @types/node --location=global
```

Comment:
9.8.1 is the latest supported for nodejs 18?

Reply:
No, the new version of Nodejs 18 is 18.20.4. It can install npm v10.x.x